lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -22,8 +24,8 @@ function divideByGCD(fraction) {
     return Node.Status.noChange(fraction);
   }
 
-  const numeratorValue = parseInt(fraction.args[0].eval());
-  const denominatorValue = parseInt(fraction.args[1].eval());
+  const numeratorValue = fraction.args[0].eval();
+  const denominatorValue = fraction.args[1].eval();
```

Comment:
the numerator and denominators are both integers that we can call `gcd` on though, yeah?